package.json

```diff
@@ -85,10 +85,10 @@
   },
   "scripts": {
     "start-local": "./scripts/serve-local.sh",
-    "start-joplin-prod": "NODE_PATH=./src CMS_API='https://joplin.herokuapp.com/api/graphql' CMS_MEDIA='https://joplin3-austin-gov-static.s3.amazonaws.com/staging/media' CMS_DOCS='https://joplin3-austin-gov-static.s3.amazonaws.com/production/media/documents' npm-run-all -p watch-css start-js",
-    "start-joplin-staging": "NODE_PATH=./src CMS_API='https://joplin-staging.herokuapp.com/api/graphql' CMS_MEDIA='https://joplin3-austin-gov-static.s3.amazonaws.com/staging/media' CMS_DOCS='multiple' npm-run-all -p watch-css start-js",
-    "start-joplin-pr": "NODE_PATH=./src CMS_API='https://joplin-pr-demo.herokuapp.com/api/graphql' CMS_MEDIA='https://joplin3-austin-gov-static.s3.amazonaws.com/staging/media' CMS_DOCS='multiple' npm-run-all -p watch-css start-js",
-    "start-joplin-local": "NODE_PATH=./src CMS_API='http://127.0.0.1:8000/api/graphql' CMS_MEDIA='http://127.0.0.1:8000/media' CMS_DOCS='multiple' npm-run-all -p watch-css start-js",
+    "start-joplin-prod": "export $(cat prod.env) && NODE_PATH=./src npm-run-all -p watch-css start-js",
```

Comment:
`source prod.env && ...` should get your variables in scope. That way we'll still be able to have comments and composable env vars within a .env file.